config/common/errors.go

```diff
@@ -52,4 +52,5 @@ var (
 	ErrNameRequired      = errors.New("metadata.name is required")
 	ErrRoleRequired      = errors.New("machineconfiguration.openshift.io/role label is required")
 	ErrInvalidKernelType = errors.New("must be empty, \"default\", or \"realtime\"")
+	ErrMCOUnsupported    = errors.New("Machine Config Operator will not manage this resource after Ignition configures it")
```

Comment:
```suggestion
	ErrMCOUnsupported    = errors.New("machine Config Operator will not manage this resource after Ignition configures it")
```

Reply:
The idea here was to keep the capitalization of the proper noun.